torch/csrc/jit/ir.h

```diff
@@ -1043,7 +1043,7 @@ struct Graph {
   TORCH_API Node* createNone(
       TypePtr typ); // value of None with type Optional[typ]
   TORCH_API Node* createAutogradZero();
-  TORCH_API Node* createFusionGroup();
+  TORCH_API Node* createFusionGroup(Symbol kind = prim::FusionGroup);
```

Comment:
Since this is no longer specific to `FusionGroup`, maybe it would be better to rename it to `createWithSubgraph(Symbol kind)`?